From af9d7dd30b0871ca845229c6f182271f3479319f Mon Sep 17 00:00:00 2001
From: Dmitry Ornatsky <cdo021c@motorola.com>
Date: Wed, 30 Sep 2015 07:54:07 -0500
Subject: [PATCH 338/959] IKKRNBSP-3423 configs: enable leds_atc driver

This is small driver to provide sysfs access to the
charging/sol LED on PMIC.

Change-Id: Ibc0103096fa97ae5a76e5085226cb1b8a6aae8d4
Signed-off-by: Dmitry Ornatsky <cdo021c@motorola.com>
Reviewed-on: http://gerrit.mot.com/787239
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 3ee2894a48f..1649ecfc181 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -50,3 +50,4 @@ CONFIG_MMI_UNIT_INFO=y
 CONFIG_MSM_WATCHDOG_CTX_PRINT=y
 CONFIG_MSM_TZ_LOG=y
 CONFIG_MSM_TZ_LOG_WDOG_DUMP=y
+CONFIG_LEDS_ATC=y
-- 
2.11.0

